### PR TITLE
Jupyter notebook eval

### PR DESCRIPTION
**Pull request by thejerrytan**
Added a jupyter notebook to visualize data, and do sanity check for trained models.

This is not ready to be merged yet.

I am including it first because it may be helpful to some of you who wants to evaluate your trained DL models.

Going to use it to generate images for README later also.

I think i did sth stupid, i commited Vatsal's dataset - TFrecord files into the repo, 266MB, ~ 1000 files. Let me know if this is bloating up your githistory and i can rewrite the githistory.

### CODE REVIEW

**Review by ser94mor — CHANGES_REQUESTED** [review]
Indeed, please, rewrite the commit history. We should not include into this repo any dataset.

**Comment by ser94mor** [comment]
This pull request **introduces 1 alert** when merging 83de649c7fc0f5201e2e2f5b94b63d91b011e365 into ed7da4b3a9eccae23995ba40593c626d1bc79cd2 - [view on LGTM.com](https://lgtm.com/projects/g/aoool/self-driving-car-using-ros/rev/pr-bba9225d52affa0fd12ad0e39132af01536d21c3)

**new alerts:**

* 1 for Unused local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by thejerrytan** [comment]
> Indeed, please, rewrite the commit history. We should not include into this repo any dataset.

this is what i did:
`git filter-branch --tree-filter 'rm -rf path/to/your/file' HEAD`
`git push --force origin jupyter-notebook-eval`

in order to remove those files from commit history.

Now to get the validation set on your own:
1) download from this link: https://github.com/coldKnight/TrafficLight_Detection-TensorFlowAPI#get-the-dataset
2) copy paste to ./data folder
3) the downloaded file name is dataset-sdcnd-capstone.zip. unzip. 
root folder is dataset-sdcnd-capstone
4) assuming you are at projectRoot/data: mv dataset-sdcnd-capstone/data ../validation

**Comment by ser94mor** [comment]
Ah, something bad happens with the diff. Could you create a new branch from master and put an `eval.ipynb` file there and create a new PR? Looks like not only your commit was rewritten.

**Comment by thejerrytan** [comment]
@aoool  this branch is closed, please refer to https://github.com/aoool/self-driving-car-using-ros/pull/14 for the new one.
